build.properties

```
@@ -5,8 +5,8 @@ core.dir = ../portapps
 app = nextcloud
 app.name = Nextcloud
 app.type = msi
-app.version = 3.2.2
-app.release = 14
+app.version = 3.4.1
+app.release = 16
```

```suggestion
app.release = 15
```

Actually should be `15`